src/OpenLoco/src/Map/Track/Track.cpp

```diff
@@ -336,7 +336,7 @@ namespace OpenLoco::Map
         setBaseZ(baseZ);
         setClearZ(clearZ);
         _type |= direction & 0x3;
-        _flags = quarterTile & 0xF;
+        _flags = static_cast<ElementFlags>(quarterTile) & ElementFlags::invalid; //The flags only occupy the top 4 bits of quarterTile
```

Comment:
This is effectively getting rid of the flags altogether, and only keeping the non-flags info. The variable name might be a bit of a misnomer for this element type.

Reply:
@AaronVanGeffen is this more clear?
_flags = static_cast<ElementFlags>(quarterTile & 0xF); //Clear the flags (top four bits of quarterTile)

Reply:
Its probably best to implement getter/setter that splits the single field then we can properly separate the flags as well.

Reply:
Added a ```clearFlags()``` function. That seemed like the most clear and safe soltuion